static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA&shortname=GET_SHORTNAME_FROM_ZETA
+endpointCompression: gzip
+maintainer:
+  email: DL-Zeta-SSP@zetaglobal.com
+gvlVendorID: 833
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+  site:
+    mediaTypes:
+      - banner
+      - video
+userSync:
+  redirect:
+    url: https://ssp.disqus.com/redirectuser?gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&r={{redirect_url}}&partner=GET_SHORTNAME_FROM_ZETA
+    uid-macro: 'BUYERUID'
```

Comment:
This doesn't seem to be correct and breaks the prebid-server code to build. You might want to correct `{{gdpr}}` with `{{.GDPR}}` 
Please also be sure that your cookie sync url work as expected and sets the correct macro.
You might also wanna replace `uid-macro` with `userMacro` and define its value as `"${BUYERUID}"`.

Reply:
Sorry, it was copy-pasted from Java adapter. Fixed.